rmf_fleet_adapter/include/rmf_fleet_adapter/agv/EasyFullControl.hpp

```diff
@@ -175,7 +175,7 @@ class EasyFullControl::Configuration
   ///   The URI for the websocket server that receives updates on tasks and
   ///   states. If nullopt, data will not be published.
   Configuration(
-    const std::string& fleet_name,
+    std::string fleet_name,
```

Comment:
Why is this a dangling reference? It previously allowed strings to be passed as both l and r values?